packages/runtime/src/index.js

```
@@ -1,47 +1,64 @@
+            })
+        } catch (error) {
+          console.error(error)
+        }
+      }
+    }
+  }
+  let etiquetasH = document.getElementsByTagName('div')
+  for (var u = 0; u < etiquetasH.length; u++) {
+    if (
+      etiquetasH[u].hasAttribute('type') &&
+      (etiquetasH[u].getAttribute('type') == 'text/eshtml' ||
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)